tests/SarTests.cpp

```diff
@@ -48,12 +48,12 @@ TEST_F(SarSensorModel, Center) {
   csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
   // TODO these tolerances are bad
   EXPECT_NEAR(groundPt.x, 1737391.90602155, 1e-2);
-  EXPECT_NEAR(groundPt.y, 3749.98835331, 1e-2);
+  EXPECT_NEAR(groundPt.y, -3749.98835331, 1e-2);
```

Comment:
These negations are because the sensor was hard-coded to left looks and the test image is a right look.

Reply:
Is the hard-coding to a left look something that will get fixed in a future PR?

Reply:
It did absolutely look like the left vs. right look was being handled, so just an old comment makes more sense!